README.md

```diff
@@ -0,0 +1,453 @@
+[![](https://travis-ci.org/buildo/avenger.svg)](https://travis-ci.org/buildo/avenger)
+[![](https://img.shields.io/npm/v/avenger.svg?sytle=flat-square)](https://www.npmjs.com/package/avenger)
+[![npm downloads](https://img.shields.io/npm/dm/avenger.svg?style=flat-square)](https://www.npmjs.com/package/avenger)
+[![](https://david-dm.org/buildo/avenger.svg)](https://david-dm.org/buildo/avenger#info=dependencies&view=list)
+[![](https://david-dm.org/buildo/avenger/dev-status.svg)](https://david-dm.org/buildo/avenger#info=devDependencies&view=list)
+
+---
+
+# Intro
+
+Avenger is a data fetching and caching layer written in TypeScript, its API is designed to mirror the principles of **Command Query Responsibility Segregation** thus of facilitating their adoption.
```

Comment:
...in TypeScript **dot** Its API is designed to mirror the principles of Command Query Responsibility Segregation ~~thus of facilitating~~  **and to facilitate** their adoption.